portal-frontend/src/app/shared/owner-dialogs/parcel-owners/parcel-owners.component.ts

```diff
@@ -92,13 +93,17 @@ export class ParcelOwnersComponent {
       });
     }
     dialog.beforeClosed().subscribe((updatedUuid) => {
-      if (updatedUuid) {
+      if (updatedUuid?.ownerDeleted) {
+        this.onOwnersDeleted.emit(updatedUuid.deletedOwnerId);
+      }
+      else if (updatedUuid) {
         this.onOwnersUpdated.emit();
       }
     });
     dialog.afterClosed().subscribe((result) => {
```

Comment:
Is there a reason we need both onBeforeClosed and onAfterClosed? I recommend you only use one.

Reply:
i just kept afterclosed

Reply:
perfect!